core/sql/executor/ExStats.cpp

```diff
@@ -2357,11 +2357,19 @@ Int64 ExHdfsScanStats::getNumVal(Int32 i) const
 
 NABoolean ExHdfsScanStats::filterForSEstats(struct timespec currTimespec, Lng32 filter)
```

Comment:
The filter is in seconds and comes from passing the user command. The user command is something like
select * from table(statistics(null, 'SE_OFFENDER=<t1_in_secs>)).  The parsing of the second parameter should take care of this.

The blockTime_ is currently in 32bit, because we don't expect the query to be blocked so long. It should timeout much earlier than this from hbase.